test/aria/utils/ArrayTestCase.js

```diff
@@ -237,6 +237,46 @@ Aria.classDefinition({
                 result += value;
             }, this);
             this.assertTrue(result === "123459");
+        },
+
+        testEvery : function () {
```

Comment:
You are testing `every` for a suitable use case which is to break from a loop, but one of its more generic and first place use cases is to return a meaningful (boolean) value. Please test it.